### ISSUE TITLE
Scripting friendly search

### ISSUE THREAD

**Issue by GrimSqueaker** [conan-io/conan#1225]
For getting detailed information about packages I would like to do something like 
```
for i in `conan search -r $SERVER`; do 
  conan search -r $SERVER $i
done
```

But conan search yields "Existing package recipes:" as well as a blank line before listing the recipes.

**Comment by memsharded** [comment]
I guess such first two lines could be removed without big problems (what if someone already built a script that skipped the first two lines?) Maybe it could be considered a more general solution that would be to let conan provide json output for certain commands, that would be much more robust to changes in output, and more convenient to parse?

**Comment by GrimSqueaker** [comment]
Yes, `conan search -r $SERVER | tail -n +3` is a workaround. But this is error prone if changes are made to how Conan prints the output.

Could you check, if Conan is running in an interactive shell (e.g. via os.isatty) and be more verbose or less verbose for scripting? Alternatively, via a command line parameter?

**Comment by GrimSqueaker** [comment]
Parsing JSON with bash or standard Unix/Linux tools is probably more cumbersome than a plain list of strings.

**Comment by memsharded** [comment]
Yes, that is true. I mostly script in python, but agree.
The interactive shell is already checked (e.g. to enable colors), so might be an option. Also the command line parameter, but maybe it would be simpler just to remove the first two lines. I will submit this for consideration to next 0.23.

**Comment by GrimSqueaker** [comment]
Thanks.

**Comment by lasote** [comment]
It was released finally in 0.24. 
Now you can use `conan search zlib* --raw`
